app/2.0.x/admin-api.md

```diff
@@ -1,6 +1,13 @@
 ---
+#
+#  WARNING: this file was auto-generated by a script.
+#  DO NOT edit this file directly. Instead, send a pull request to change
+#  https://github.com/Kong/kong/blob/master/scripts/autodoc-admin-api
+#  or its associated files instead.
+#
 title: Admin API
 toc: false
+skip_read_time: true
```

Comment:
Added because `declarative-admin-api.md` had this, and the two files share the same template.